### DISCUSSION TITLE
How to fix We weren’t able to create the release for you. Make sure you have a valid tag.

### DISCUSSION THREAD

**Discussion by ContentLTD** [community/community#40234] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

How to fix `We weren’t able to create the release for you. Make sure you have a valid tag.  
` 
I cant  create a tag
![image](https://user-images.githubusercontent.com/103521110/204168468-f1468fcb-840d-4e86-a21f-016cb44f4ea5.png)
![image](https://user-images.githubusercontent.com/103521110/204168472-9c950ac2-260d-45fd-bf02-c08ee46c8f6b.png)

**Comment by radar-blip** [comment, marked as answer]
Try typing the tag name first in `Find or create a new tag`, and below that it should be suggested for you.
Then just click on the suggested tag name.
Should work.

![tag](https://user-images.githubusercontent.com/39196767/204196943-749c2bca-5f4b-4a5d-81e8-a7cfaa592ffd.png)

**Reply by ContentLTD** [reply]
Thank you so much. That helped

**Reply by radar-blip** [reply]
No problem. Happy to help!
